pom.xml

```diff
@@ -55,9 +55,17 @@
         <carbon.identity.version.4.5.3>4.5.3</carbon.identity.version.4.5.3>
         <carbon.multitenancy.version.4.4.1>4.4.1</carbon.multitenancy.version.4.4.1>
         <carbon.registry.version.4.4.4>4.4.4</carbon.registry.version.4.4.4>
+        <carbon.kernel.version>4.4.0</carbon.kernel.version>
```

Comment:
Since there can be multiple kernel version like 4.4.0, 4.4.1, 4.4.2, etc., it is better to mention the version as carbon.kernel.version.4.4.0.